### PR TITLE
#160056177 Set up sass pre-processing, Bootstrap 4.0 framework and update readme

### PR DESCRIPTION
**Pull request by Oluwafayokemi**
#### What does this PR do?
Set up sass processing and bootstrap on the client and update readme
#### Description of Task to be completed?
- Set up sass processing on the client
- Set up a bootstrap 4.0 framework on the client
- Update readme.md
#### How should this be manually tested?
- create a .scss file and write some styling, 
- Try using some bootstrap properties in .js file
- import the created .scss file in a .js file
- run npm start
- You should see your style being effected without any compilation error
#### What are the relevant pivotal tracker stories?
[#160056177](https://www.pivotaltracker.com/story/show/160056177)
#### Screenshots (if appropriate)
image

### CODE REVIEW

**Review by chukwuemekachm — COMMENTED** [review]
Good work @Oluwafayokemi, I'm very impressed with this PR. However, you are having quite a number of EOL squigglies in package.json, _all.scss, _color.scss, _mixins.scss, _home.scss, _normalize.scss. Please endeavor to resolve them.
Weldone.

**Comment by Oluwafayokemi** [comment]
Thank you @Chukwuemeka Chima <chukwuemekachm@gmail.com> I am on it.

**Review by madeofhuman — APPROVED** [review]
Good job setting up this, Fayo.

**Comment by Oluwafayokemi** [comment]
@Chukwuemeka Chima <chukwuemekachm@gmail.com> I have fixed the eol issues.
Thank you for the feedback.

File: src/styles/App.css [hunk 1]
```diff
@@ -0,0 +1,329 @@
+@import url("https://fonts.googleapis.com/css?family=Dancing+Script");
+body {
```
Comment (tonymontaro) [inline src/styles/App.css]: Good job @Oluwafayokemi. This file is generated (App.css), we should avoid committing generated files.

**Review by Veraclins — APPROVED** [review]

**Review by tonymontaro — APPROVED** [review]
Good Job!

**Pull request merged**